src/tests/test_utils/functional_test_utils/include/functional_test_utils/core_config.hpp

```diff
@@ -1,9 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#pragma once
-
-#include "shared_test_classes/base/layer_test_utils.hpp"
-
-void CoreConfiguration(LayerTestsUtils::LayerTestsCommon* test);
```

Comment:
Just FYI: we have new `core_configuration` for OV2.0 based class

Reply:
@iefode Yes, I know :smile: